documentation/docs/03-loading.md

```diff
@@ -13,12 +13,8 @@ export interface LoadInput<
 	Stuff extends Record<string, any> = Record<string, any>,
 	Session = any
 > {
-	page: {
-		origin: string;
-		path: string;
-		params: PageParams;
-		query: URLSearchParams;
-	};
+	url: URL;
+	params: PageParams;
```

Comment:
This needs updating.

Reply:
It does? This is the `load` arguments, not the stores — looks correct to me?

Reply:
Oh! I guess I was assuming the `load` arguments would also reflect the available stores. Are they not anymore? Is that confusing? (It was confusing to me just now, anyway.)

Reply:
see https://github.com/sveltejs/kit/pull/3126#issuecomment-1002753326

Reply:
I see. I had skimmed that, and had intended to circle back to that afterwards, but I apparently did not skim it well enough. Hm. I guess for consistency what I would argue for is a `route` argument to `load` that contains `url` and `params`, but that might be more inconvenient than it's worth. Although, if you're not worried about `$route.url.pathname`, why be worried about it in `load`?